### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by mahinsagotra**

### CODE REVIEW

File: 2-js-basics/1-data-types/README.md [hunk 1]
```diff
@@ -74,36 +76,37 @@ Constants are similar to variables, with two exceptions:
...
+    ```
+
+    Above you are changing the value of the object but not the reference itself, which makes it allowed.
+
+  > Note, a `const` means the reference is protected from reassignment. The value is not _immutable_ though and can change, especially if it's a complex construct like an object.
 
 ## Data Types
 
-Variables can store many different types of values, like numbers and text. These various types of values are known as the **data type**. Data types are an important part of software development because it helps developers make decisions on how the code should be written and how the software should run. Furthermore, some data types have unique features that help transform or extract additional information in a value.
+Variables can store many different types of values, like `numbers` and `text`. These various types of values are known as the **Data type**. Data types are an important part of software development because it helps developers make decisions on how the code should be written and how the software should run. Furthermore, some data types have unique features that help transform or extract additional information in a value.
```
Comment (jlooper) [inline 2-js-basics/1-data-types/README.md:107]: please remove this change

Comment (jlooper) [inline 2-js-basics/1-data-types/README.md:107]: please remove this edit

File: 2-js-basics/1-data-types/README.md [hunk 2]
```diff
@@ -150,24 +153,23 @@ let myString2 = "World";
...
-**Template literals** are another way to format strings, except instead of quotes, the backtick  is used. Anything that is not plain text must be placed inside placeholders `${ }`. This includes any variables that may be strings.
+**Template literals** are another way to format strings, except instead of quotes, the backtick is used. Anything that is not plain text must be placed inside placeholders `${ }`. This includes any variables that may be strings.
 
 ```javascript
 let myString1 = "Hello";
 let myString2 = "World";
 
 `${myString1} ${myString2}!` //Hello World!
-`${myString1}, ${myString2}!` //Hello World!
+`${myString1}, ${myString2}!`; //Hello World!
```
Comment (jlooper) [inline 2-js-basics/1-data-types/README.md:167]: please remove this edit

File: 2-js-basics/2-functions-methods/README.md [hunk 3]
```diff
@@ -21,27 +23,28 @@ Just as important is the ability to name a function. While this might seem trivi
...
+Whenever we want to call (or invoke) our function, we use the name of the function followed by `()`. It's worth noting the fact that our function can be defined before or after we decide to call it; the JavaScript compiler will find it for you.
 
 ```javascript
 // calling our function
 displayGreeting();
 ```
 
-> **NOTE:** There is a special type of function known as a **method**, which you've already been using! In fact, we saw this in our demo above when we used `console.log`. What makes a method different from a function is a method is attached to an object (`console` in our example), while a function is free floating. You will hear many developers use these terms interchangeably.
+> **NOTE:** There is a special type of function known as a **method**, which you've already been using! In fact, we saw this in our demo above when we used `console.log`. What makes a method different from a function is that, a `method` is attached to an `object` (`console` in our example), while a function is free floating. You will hear many developers use these terms interchangeably.
```
Comment (jlooper) [inline 2-js-basics/2-functions-methods/README.md:47]: please remove this edit

File: 2-js-basics/2-functions-methods/README.md [hunk 4]
```diff
@@ -53,14 +56,12 @@ There are a handful of best practices to keep in mind when creating functions
 
 ## Passing information to a function
 
-To make a function more reusable you'll often want to pass information into it. If we consider our `displayGreeting` example above, it will only display **Hello, world!**. Not the most useful function one could create. If we want to make it a little more flexible, like allowing someone to specify the name of the person to greet, we can add a **parameter**. A parameter (also sometimes called an **argument**), is additional information sent to a function.
+To make a function more reusable you'll often want to pass information into it. If we consider our `displayGreeting` example above, it will only display **Hello, world!**. Not the most useful function one could create. If we want to make it a little more flexible, like allowing someone to specify the name of the person to greet, we can add a **parameter**. A Parameter (also sometimes called an **argument**), is additional information sent to a function.
```
Comment (jlooper) [inline 2-js-basics/2-functions-methods/README.md:59]: please remove this edit

File: 2-js-basics/2-functions-methods/README.md [hunk 5]
```diff
@@ -53,14 +56,12 @@ There are a handful of best practices to keep in mind when creating functions
...
 
 Parameters are listed in the definition part within parenthesis and are comma separated like so:
 
 ```javascript
-function name(param, param2, param3) {
-
-}
+function name(param1, param2, param3) {} //param short for parameter
```
Comment (jlooper) [inline 2-js-basics/2-functions-methods/README.md:64]: please remove this edit

File: 2-js-basics/2-functions-methods/README.md [hunk 6]
```diff
@@ -151,22 +151,22 @@ When we are passing a function as a parameter we can bypass creating one in adva
...
+  console.log("3 seconds has elapsed");
 });
 ```
 
 If you run our new code you'll notice we get the same results. We've created a function, but didn't have to give it a name!
 
 ### Fat arrow functions
 
-One shortcut common in a lot of programming languages (including JavaScript) is the ability to use what's called an **arrow** or **fat arrow** function. It uses a special indicator of `=>`, which looks like an arrow - thus the name! By using `=>`, we are able to skip the `function` keyword.
+One shortcut which is common in a lot of programming languages (including JavaScript) is the ability to use what's called an **arrow** or **fat arrow** function. It uses a special indicator of `=>`, which looks like an arrow - thus the name! By using `=>`, we are able to skip the `function` keyword.
```
Comment (jlooper) [inline 2-js-basics/2-functions-methods/README.md:163]: please remove this edit

**Review by jlooper — CHANGES_REQUESTED** [review]
thank you for the edits, please roll a few of them back, thanks

**Comment by mahinsagotra** [comment]
So, you want me to remove all the changes? @jlooper

**Comment by jlooper** [comment]
@mahinsagotra only the ones flagged, please.

**Comment by mahinsagotra** [comment]
@jlooper except the index.html changes, it's showing all other changes flagged

**Pull request closed without merging**